CRVResponse/src/MakeCrvWaveforms.cc

```diff
@@ -63,14 +63,15 @@ void MakeCrvWaveforms::MakeWaveform(const std::vector<double> &times,
   {
     double timeOfCharge=*iterTime;  //the time when the charge happened
     double charge=*iterCharge/_singlePEReferenceCharge;  //scale it to the 1PE reference charge used for the single PE waveform
-    unsigned int waveformIndex = (timeOfCharge>startTime ? ceil((timeOfCharge-startTime)/digitizationPrecision) : 0);  //waveform index of the first digitization point for this particular charge
+    if(timeOfCharge<startTime) continue; //this shouldn't happen
```

Comment:
Does this comment mean that physically it shouldn't happen?  Or that this condition signals an error in the code?

Reply:
The function that calls this function makes sure that it doesn't happen. So it means it's an error, if it happens. I will change the code to throw an exception here.